src/components/Header/Header.css

```diff
@@ -7,8 +7,8 @@
   text-transform: uppercase;
   background-color: var(--main-bg-color);
   font-family: "Montserrat", sans-serif;
-  font-size: 3vw;
-  font-weight: 900;
+  font-size: 5.5vw;
+  /* font-weight: 900; */
```

Comment:
you can delete this :)

Reply:
Hey Rikki, many thank for reviewing. Cool that you took the time. I will delete that.